app/models/obituary.rb

```diff
@@ -1,4 +1,10 @@
 class Obituary < ApplicationRecord
   validates_presence_of :first_name, :last_name
   belongs_to :user
+  before_save :default_image
+
+  private
+  def default_image
+    self.image_url ||= 'https://image.shutterstock.com/image-vector/ui-image-placeholder-wireframes-apps-260nw-1037719204.jpg'
+  end
```

Comment:
Sets a placeholder image for an obituary before if there isn't one already.

Reply:
Good idea

Reply:
I started working on a logo. Maybe we can set this as the logo once we decide on one!